### PR TITLE
S749 allow whitespace layer names for vizjson

### PR DESCRIPTION
**Pull request by juanignaciosl**
We allow setting a whitespace as layer name, but in vizjson 3 it got replaced by the table name, making it impossible to hide the name at layers selector. This PR makes the vizjson (and the layer selector) display the empty space, simulating no name).

### CODE REVIEW

File: app/services/carto/visualizations_export_service_2.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 require 'json'
-require 'carto/export/layer_exporter'
+require_dependency 'carto/export/layer_exporter'
```
Comment (javitonino) [inline app/services/carto/visualizations_export_service_2.rb]: 🤔  This is a revert of a change you did to fix rakes, see: https://github.com/CartoDB/cartodb/commit/a433e8fdd68f80dfbd9f20e5592bee8c61acae7f

IIRC, this broke during `rake db:migrate` and we had to fall back to `require`.

Comment (juanignaciosl) [inline app/services/carto/visualizations_export_service_2.rb]: Yep, but this commit should've fixed it: 794a96e37edd620655c7df937ccec9424e9e6b36 . Have you reproduced the issue?

Comment (javitonino) [inline app/services/carto/visualizations_export_service_2.rb]: I'm blind 🙈 

No, I have not reproduced it, and the fix looks great, thanks for that 💋 We'll make sure to double-check in acceptance that rakes run correctly, but it looks promising :)

**Comment by javitonino** [comment]
Hah, what a hack to use blank space names for this, but with these changes, it works! Also, we didn't break rakes, so that's a plus :D

Deploying...

**Pull request merged**